pypeit/tests/test_flux.py

```diff
@@ -109,3 +110,15 @@ def test_extinction_correction():
     np.testing.assert_allclose(flux_corr[0], 4.47095192)
 
 
+def test_filter_scale():
+    # Test scale_in_filter() method which is called in coadding
+    wave = np.arange(3000.,10000.)
+    flux = np.ones_like(wave)
+    gdm = np.ones_like(wave, dtype=bool)
+    #
+    par = Coadd1DPar()
+    par['filter'] = 'DECAM-R'
+    par['filter_mag'] = 17.
+    # Run
+    scale = flux_calib.scale_in_filter(wave, flux, gdm, par)
```

Comment:
Does this need to take the full parameter set? I.e., can we just pass in `filter` and `filter_mag`?

Reply:
we need `mag_type` too and `filter_mask`.
I drew the line at 4.  ;)